components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -929,7 +943,12 @@ interface WritableHistoryMetadataConnection : ReadableHistoryMetadataConnection
     /**
      * Record or update metadata information about a URL. See [HistoryMetadataObservation].
      */
-    suspend fun noteHistoryMetadataObservation(key: HistoryMetadataKey, observation: HistoryMetadataObservation)
+    suspend fun noteHistoryMetadataObservation(observation: HistoryMetadataObservation)
```

Comment:
Not sure we need to expose the generic version, we don't have anything that'll use it afaik. But I suppose it doesn't hurt.